### PR TITLE
insert fadeInImages

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: shared/Button.tsx [hunk 1]
```diff
@@ -19,15 +19,15 @@ interface ToggleButtonProps {
   value?: boolean
 }
 
-export const Button: FC<ButtonProps> = ({ children, className, onClick, ...rest }) => {
+export const Button = ({ children, className, onClick, ...rest }: ButtonProps) => {
   const click = () => {
```
Comment (crazzywizard) [inline shared/Button.tsx:24]: Why did we remove `FC<ButtonProps>`?

Comment (towardsnewtech) [reply shared/Button.tsx:23]: While fixing issues style flipping, it will be removed.

File: components/Chat/components/ChatBox/ChatBox.tsx [hunk 2]
```diff
@@ -33,15 +33,15 @@ const ChatBox = () => {
               <MyMessage
                 message={JSON.parse(message.data)}
                 key={message}
-                connectionId={message.connectionId}
+                // connectionId={message.connectionId}
```
Comment (sweetmantech) [inline components/Chat/components/ChatBox/ChatBox.tsx]: please remove commented code 🚮 
```suggestion
```

File: components/Chat/components/ChatBox/ChatBox.tsx [hunk 3]
```diff
@@ -33,15 +33,15 @@ const ChatBox = () => {
...
               />
             )
           }
           return (
             <OtherMessage
               message={JSON.parse(message.data)}
               key={message}
-              connectionId={message.connectionId}
+              // connectionId={message.connectionId}
```
Comment (sweetmantech) [inline components/Chat/components/ChatBox/ChatBox.tsx]:
```suggestion
```

Comment (sweetmantech) [reply components/Chat/components/ChatBox/ChatBox.tsx]: please remove commented code

@crazzywizard do we need this?

File: components/Chat/components/ChatBox/ChatBox.tsx (hunk 2 above)
Comment (sweetmantech) [reply components/Chat/components/ChatBox/ChatBox.tsx]: why are you commenting this? 

is it required?

File: components/Chat/components/ChatBox/ChatBox.tsx (hunk 3 above)
Comment (crazzywizard) [reply components/Chat/components/ChatBox/ChatBox.tsx]: i pass that for identification only.

File: components/LandingPage/sections/Networking.tsx [hunk 4]
```diff
@@ -86,40 +145,76 @@ const Networking = () => {
...
+                ))
+              : imagesList.map((imageData: ImageData) => (
+                  <div key={imageData.key} className="flex justify-center items-center">
+                    <FadeInImage
+                      url={themeMode === "light" ? imageData.image : imageData.white_image}
+                      width={imageData.width}
+                      height={imageData.height}
+                    />
+                  </div>
+                ))}
```
Comment (crazzywizard) [inline components/LandingPage/sections/Networking.tsx]: why two separate arrays for mobile and desktop?

**Pull request merged**